### DISCUSSION TITLE
Automated Export configuration

### DISCUSSION THREAD

**Discussion by StreamFacility** [bitfocus/companion#3073] [Ideas]
I've been thinking about a really helpful feature for the companion software: automated configuration export. It would be great to have the option to automatically export your configuration to a specific location. This could be triggered periodically (e.g., daily, weekly) or even based on specific events (button press, specific condition).

This would be super useful for:

Backup: Having regular backups of your configuration would help you recover from accidental changes or data loss.
Remote access to the config file: If the export is saved to a shared drive, the config can be locally reiewed without having to have access to the remote companion installation

**Comment by johnsturgeon** [comment]
Yes Please!
